pkg/cmd/openshift-network-controller/network_controller.go

```diff
@@ -0,0 +1,83 @@
+package openshift_network_controller
+
+import (
+	"os"
+
+	"github.com/golang/glog"
+
+	"k8s.io/api/core/v1"
+	"k8s.io/client-go/kubernetes"
+	v1core "k8s.io/client-go/kubernetes/typed/core/v1"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/leaderelection"
+	"k8s.io/client-go/tools/leaderelection/resourcelock"
+	"k8s.io/client-go/tools/record"
+	"k8s.io/kubernetes/pkg/api/legacyscheme"
+
+	openshiftcontrolplanev1 "github.com/openshift/api/openshiftcontrolplane/v1"
+	"github.com/openshift/origin/pkg/cmd/openshift-controller-manager"
+	origincontrollers "github.com/openshift/origin/pkg/cmd/openshift-controller-manager/controller"
+	"github.com/openshift/origin/pkg/cmd/util"
+)
+
+func RunOpenShiftNetworkController(config *openshiftcontrolplanev1.OpenShiftControllerManagerConfig, clientConfig *rest.Config) error {
+	util.InitLogrus()
+	kubeClient, err := kubernetes.NewForConfig(clientConfig)
+	if err != nil {
+		return err
+	}
+
+	originControllerManager := func(stopCh <-chan struct{}) {
+		if err := openshift_controller_manager.WaitForHealthyAPIServer(kubeClient.Discovery().RESTClient()); err != nil {
+			glog.Fatal(err)
+		}
+
+		controllerContext, err := origincontrollers.NewControllerContext(*config, clientConfig, nil)
+		if err != nil {
+			glog.Fatal(err)
+		}
+		enabled, err := origincontrollers.RunSDNController(controllerContext)
```

Comment:
can I remove this from `hypershift openshift-controller-manager`?

Reply:
I believe that removing it from `pkg/cmd/openshift-controller-manager/controller/config.go` would remove it from both operator-based and ansible-based installs and thus break current CI, but if it's possible to remove `"openshift.io/sdn"` from the default set of controllers on operator-based installs only then we can do that

Reply:
> I believe that removing it from `pkg/cmd/openshift-controller-manager/controller/config.go` would remove it from both operator-based and ansible-based installs and thus break current CI, but if it's possible to remove `"openshift.io/sdn"` from the default set of controllers on operator-based installs only then we can do that

Ok, I'll do that after this merges.